### DISCUSSION TITLE
после переноса нет картинок

### DISCUSSION THREAD

**Discussion by jorjik** [maxsite/cms#489] [Q&A]
subj.

Version 107.5
в консоле так https://i.imgur.com/PDmv8rd.png
PHP 7.4

**Comment by maxsite** [comment]
После какого переноса? Картинку прямо сюда загрузите (сайт недоступен).

**Comment by jorjik** [comment]
*после переноса на другой хостинг
скрин 
![PDmv8rd](https://user-images.githubusercontent.com/116784/154999184-a52dc6a8-cae1-45af-81c5-5856eef763ee.png)

**Comment by maxsite** [comment]
А сами изображения доступны через браузер по прямой ссылке?

**Comment by jorjik** [comment]
они есть в админке (и по ссылке там же)

**Comment by maxsite** [comment]
Нужно тот адрес, который в поле Изображения записи. И проверьте может нужно права на каталоги uploads — это если миниатюры не создаются.

**Comment by jorjik** [comment]
раздал права; картинки не обычные!
`<img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAQQAAAC+AQMAAADp4OPuAAAABlBMVEXu7u53d3fXzPnzAAAACXBIWXMAAA7EAAAOxAGVKw4bAAAAHUlEQVRYhe3BMQEAAADCoPVP7WULoAAAAAAAALgBGTwAAdMWDJ8AAAAASUVORK5CYII=" class="w100" alt="Мультибраузеры (Антидетект)">`

**Comment by maxsite** [comment]
Это плейсхолдер/заглушка, который формируется, если файл миниатюры недоступен. А) проверьте что изображение записи по прямой ссылке. Б) Если файл доступен, то смотрите каталог этой записи вв uploads/_pages и там должны быть миниатюры.

Скорее всего просто нет прав на запись (777) на каталог миниатюр. Обычно достаточно только на uploads, но иногда нужно прописывать и на все подкаталоги. Так настроен сервер и за это нужно пинать админа. Может неверн определён владелец каталога (не php, а юзер). Опять — пинать админа.
